packages/zoe/src/types.js

```diff
@@ -169,148 +157,93 @@
  * `{ afterDeadline: { timer :Timer<Deadline>, deadline :Deadline } }
  */
 
-/**
- * @template OC - the offer outcome
- * @typedef {Payment & { _inviteOutcome: OC }} Invite
- * An invitation to participate in a Zoe contract.
- * Invites are Payments, so they can be transferred, stored in Purses, and
- * verified. Only Zoe can create new Invites.
- */
-
-/**
- * @template OC - the offer outcome
- * @callback MakeContract The type exported from a Zoe contract
- * @param {ContractFacet} zcf The Zoe Contract Facet
- * @returns {Invite<OC>} invite The closely-held administrative invite
- */
-
 /**
  * @typedef {Object} ContractFacet
  * The Zoe interface specific to a contract instance.
  * The Zoe Contract Facet is an API object used by running contract instances to
  * access the Zoe state for that instance. The Zoe Contract Facet is accessed
  * synchronously from within the contract, and usually is referred to in code as
  * zcf.
- * @property {Reallocate} reallocate Propose a reallocation of extents per offer
- * @property {Complete} complete Complete an offer
+ * @property {Reallocate} reallocate - reallocate amounts among seats
+ * @property {SaveIssuer} saveIssuer - save an issuer to ZCF and Zoe
+ * and get the amountMath and brand synchronously accessible after saving
  * @property {MakeInvitation} makeInvitation
- * @property {AddNewIssuer} addNewIssuer
- * @property {InitPublicAPI} initPublicAPI
+ * @property {Shutdown} shutdown
  * @property {() => ZoeService} getZoeService
- * @property {() => Issuer} getInviteIssuer
- * @property {(offerHandles: OfferHandle[]) => OfferStatus} getOfferStatuses
- * @property {(offerHandle: OfferHandle) => boolean} isOfferActive
- * @property {(offerHandles: OfferHandle[]) => OfferRecord[]} getOffers
- * @property {(offerHandle: OfferHandle) => OfferRecord} getOffer
- * @property {(offerHandle: OfferHandle, brandKeywordRecord?: BrandKeywordRecord) => Allocation} getCurrentAllocation
- * @property {(offerHandles: OfferHandle[], brandKeywordRecords?: BrandKeywordRecord[]) => Allocation[]} getCurrentAllocations
- * @property {(offerHandle: OfferHandle) => Promise<Notifier<Allocation|undefined>>} getOfferNotifier
- * @property {() => InstanceRecord} getInstanceRecord
+ * @property {() => invitationIssuer} getInvitationIssuer
+ * @property {() => InstanceRecord } getInstanceRecord
  * @property {(issuer: Issuer) => Brand} getBrandForIssuer
```

Comment:
I was lazy in typing presences as the same as the underlying object.  Ideally, a presence type would only be unwrapped and rewrapped by E() but I did not get a round tuit.

Reply:
Also note that presence types would only be used in the remote client, not the local vat's type since it is a constraint imposed by the Marshal stack.

Reply:
That's why a type meaning `{T | Presence<T>}` would be more useful than a type meaning `{Presence<T>}`. The former means the actual object may or may not be remote, but in either case
   * the object identity is accurate
   * if used as an argument in a message to something co-located with the actual object, the actual object arrives.